control_panel_api/permissions.py

```diff
@@ -28,5 +31,23 @@ class S3BucketPermissions(IsSuperuser):
     pass
 
 
-class UserPermissions(IsSuperuser):
+class IsSelf(BasePermission):
+
+    def has_permission(self, request, view):
+        if super().has_permission(request, view):
```

Comment:
Do we need to check the parent in this case? `BasePermission#has_permission()` always returns `True`, it may be a bit overkill.

Reply:
The parent will be `IsSuperuser`

Reply:
This is obviously more complicated than it seems.
I can now see why you were checking `super()`. Annoyingly as far as I can tell from the DRF documentation, if any of the permissions classes in the chain returns `False` or an exception it doesn't check the rest of the classes which in our case may be handy.

Reply:
My implementation is unnecessarily complicated - I will refactor it.